frontend/src/lib/components/ChartFilter/ChartFilter.tsx

```diff
@@ -28,15 +29,18 @@ export function ChartFilter({ filters, onChange, disabled }: ChartFilterProps):
     const { chartFilter } = useValues(chartFilterLogic(insightProps))
     const { setChartFilter } = useActions(chartFilterLogic(insightProps))
 
+    const seriesCount = toLocalFilters(filters).length
     const cumulativeDisabled = filters.insight === InsightType.STICKINESS || filters.insight === InsightType.RETENTION
-    const pieDisabled: boolean = filters.insight === InsightType.RETENTION || filters.insight === InsightType.STICKINESS
+    const pieDisabled: boolean = filters.insight === InsightType.STICKINESS || filters.insight === InsightType.RETENTION
     const worldMapDisabled: boolean =
-        filters.insight === InsightType.RETENTION ||
         filters.insight === InsightType.STICKINESS ||
-        (!!filters.breakdown &&
+        (filters.insight === InsightType.RETENTION &&
```

Comment:
retention used to be outside of this check

so it was retention OR stickiness OR there are non-country breakdowns

now it's stickiness OR (retention AND non country breakdown) OR more than one series

is that right?

should it be stickiness OR retention OR non country breakdown OR more than one series

Reply:
Yes, thanks for catching this!